readme.md

```diff
@@ -381,6 +406,9 @@ for future friendly performant code is to always prefer object literals. This su
 since we recommend returning object literals from functions (rather than using classes
 or constructors) as a general best coding practice.
 
+_Edit: Jakob Kummerow noted in [http://disq.us/p/1kvomfk]() that Turbofan
+can optimize away the object allocation in this specific microbenchmark.
+We are working on a updated version to take this in consideration._
```

Comment:
We are working on a updated version to take this in consideration -> We'll be updating again soon to take this into account